README.md

```diff
@@ -128,11 +128,12 @@ Start by specifying that you will be using an external configuration file:
     config-file: './.github/dependency-review-config.yml'
 ```
 
-And then create the file in the path you just specified:
+And then create the file in the path you just specified. Please note
+that the **option names in external files use underscores instead of dashes**:
```

Comment:
🤔 I assume there's some known friction involved 👍  but I wonder could we relax the option name parsing to accept both formats? Feels like this will be tough for end users to remember

Reply:
Agreed, it's on the TODO. This PR only fixes the bug with defaults value, but going forward I think supporting both options by replacing `-` with `_` when found is the way.